src/content/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx

```diff
@@ -81,90 +81,101 @@ To install our Kubernetes integration, we recommend that you follow the instruct
     <Steps>
       <Step>
 
-## Prepare to install
+        ## Prepare to install
 
-Prepare your Kubernetes system for the guided install:
+        Prepare your Kubernetes system for the guided install:
 
-* If custom manifests have been used instead of Helm, you will need to first remove the old installation using `kubectl delete -f previous-manifest-file.yml`, and then proceed through the guided installer again. This will generate an updated set of manifests that can be deployed using `kubectl apply -f manifest-file.yml`.
-* Make sure you're using the supported Kubernetes versions and make sure to check out the preliminary notes for your managed services or platforms on our [compatibility and requirements page](/docs/kubernetes-pixie/kubernetes-integration/get-started/kubernetes-integration-compatibility-requirements).
-* Make sure you have your New Relic <InlinePopover type="licenseKey" />. You can set up a free account&mdash;no credit card required.
-* Make sure the newrelic dockerhub (`https://hub.docker.com/u/newrelic`) and Google's registry (`registry.k8s.io`) domains are added to your allow list. This is where the installation will pull container images from. Note, you may need to follow the [commands](https://kubernetes.io/blog/2023/03/10/image-registry-redirect/#how-can-i-check-if-i-am-impacted) to identify the additional Google registry domains to be added to your white list, because `registry.k8s.io` typically redirects to your local registry domain (e.g., `asia-northeast1-docker.pkg.dev`) based on your region.
+        * If custom manifests have been used instead of Helm, you will need to first remove the old installation using `kubectl delete -f previous-manifest-file.yml`, and then proceed through the guided installer again. This will generate an updated set of manifests that can be deployed using `kubectl apply -f manifest-file.yml`.
+        * Make sure you're using the supported Kubernetes versions and make sure to check out the preliminary notes for your managed services or platforms on our [compatibility and requirements page](/docs/kubernetes-pixie/kubernetes-integration/get-started/kubernetes-integration-compatibility-requirements).
+        * Make sure you have your New Relic <InlinePopover type="licenseKey" />. You can set up a free account&mdash;no credit card required.
```

Comment:
I think starting three bullets with "Make sure" delays the reader. Can you cut that phrase or rewrite it so it's not repeated so frequently?